Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -492,6 +510,22 @@ - (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSe
     return asset;
 }
 
+- (UIView *)dequeueReusableOverlayView
+{
+    UIView *view = [self.reusableOverlayViews lastObject];
+
+    if (view) {
+        [self.reusableOverlayViews removeLastObject];
+        return view;
+    }
+
+    if (self.overlayViewClass) {
+        return [self.overlayViewClass new];
+    } else {
+        return [UIView new];
```

Comment:
We probably should throw an error if no class is configured and we need to use one.